vita3k/regmgr/src/regmgr.cpp

```diff
@@ -58,15 +59,32 @@ static std::string decryptRegistryFile(const fs::path reg_path) {
     return res;
 }
 
+enum RegType {
+    REG_TYPE_INT,
+    REG_TYPE_STR,
+    REG_TYPE_BIN,
+};
+
 struct RegValue {
     std::string name;
+    RegType type;
     uint32_t size;
-    std::string value;
+    std::vector<char> init_value;
 };
 
 static std::vector<std::string> reg_category_template;
 static std::map<std::string, std::vector<RegValue>> reg_template;
 
+static std::vector<char> convert_int_to_hex_string(const int32_t &value) {
```

Comment:
```suggestion
static std::vector<char> convert_int_to_hex_string(int32_t value) {
```
Don't pass an int by reference

Reply:
ah yes